### ISSUE TITLE
Tests fail when only `one_of` is provided

### ISSUE THREAD

**Issue by riker09** [agilepathway/label-checker#228]
This action does not work when the PR is assigned a `cicd` label. 

Here's the simplified workflow YAML: https://github.com/agilepathway/label-checker/issues/227#issuecomment-874112651

All tests are being run, this is the resulting log:

```
[...]
2021-07-06T06:29:08.0452411Z with:
2021-07-06T06:29:08.0452792Z   any_of: cicd
2021-07-06T06:29:08.0454423Z   repo_token: ***
2021-07-06T06:29:08.0454865Z ##[endgroup]
2021-07-06T06:29:08.0502532Z ##[command]/usr/bin/docker run --name ***REDACTED*** --label 8a33c1 --workdir /github/workspace --rm -e INPUT_ANY_OF -e INPUT_REPO_TOKEN -e INPUT_ONE_OF -e INPUT_NONE_OF -e INPUT_ALL_OF -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/***REDACTED***":"/github/workspace" ***REDACTED***
2021-07-06T06:29:08.4962745Z Checking GitHub labels ...
2021-07-06T06:29:08.6978831Z Label check successful: required none of , and found 0.
2021-07-06T06:29:08.6979653Z Label check successful: required any of cicd, and found 1: cicd
2021-07-06T06:29:08.6985999Z Error: Label check failed: required 1 of , but found 0.
2021-07-06T06:29:08.6986667Z Label check failed: required all of , but found 0.
```

From the looks of it I would say that even though I only passed one option (`one_of`) the other three tests are also evaluated with empty values. This looks like a bug to me and I would gladly help, but I don't know any Go.

_Originally posted by @riker09 in https://github.com/agilepathway/label-checker/issues/227#issuecomment-874503869_

**Comment by riker09** [comment]
It works when I add all options (leave them non-empty):

```yaml
    - id: check-label
      name: Check pull request labels
      uses: agilepathway/label-checker@v1.0.90
      continue-on-error: true
      with:
        any_of: cicd
        one_of: cicd
        all_of: cicd
        none_of: non-existing-label
        repo_token: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by johnboyes** [comment]
Hi @riker09, as per my [comment in 227](https://github.com/agilepathway/label-checker/issues/227#issuecomment-881857810) really sorry that I'm only replying now.  Thanks for raising this, will look into it and get back to you shortly.

**Comment by johnboyes** [comment]
Hi again @riker09, I've released a fix for this [v1.0.91](https://github.com/agilepathway/label-checker/releases/tag/v1.0.91).  Thanks for reporting it, it was caused by me wrongly expecting the checks to be not present by default, whereas they actually default to an empty string.  Please let me know if it fixes this issue, and also if  #227 is still an issue after this fix.  Thanks, John

**Comment by riker09** [comment]
Hello @johnboyes and no worries for a "late" reply. Sometimes I receive responses for issues that I've commented years ago, so you are well within the socially accepted limit. 😀 

I will check out the new version and report any results here as soon as I find the time. Vacation season is around the corner, though. If you don't hear from me then you know the reason behind it.

**Comment by johnboyes** [comment]
Hi @riker09, just following up on this to see if it did indeed fix your issue?  No probs if vacation season is in full swing, no rush 🏖️  😎

**Comment by riker09** [comment]
It did! Thanks 😃

**Comment by johnboyes** [comment]
Fixed by #230
